.ci/Jenkinsfile

```diff
@@ -344,15 +344,26 @@ def scpr(workspace, connection, remote_src, local_dst){
   }
 }
 
+def getPythonPath(version) {
+  sh(script: "curl https://pyenv.run | bash", label: "Install pyenv", returnStatus: true)
+  withEnv(["PATH+PYENV=${WORKSPACE}/.pyenv/bin"]){
```

Comment:
I'd retrieve the status and do not execute `install > global > shims` if the .pyenv dir is present.